apps/server/src/utils/InstitutionResolver.ts

```diff
@@ -0,0 +1,50 @@
+import type { InstitutionProvider, LocalInstitution, Provider, ResolvedInstitution } from '../shared/contract'
+import ElasticsearchClient from './elasticSearchClient'
+
+export async function resolveInstitutionProvider (institutionId: string): Promise<ResolvedInstitution> {
```

Comment:
This function needs some tests

Reply:
This function will change a lot but I'll add some tests for it's current state.